R/validate_per.R

```diff
@@ -20,4 +20,28 @@ validate_per <- function(per, missing_per) {
       stop("`per = '", per, "'` but must be one of 'year', 'fortnight', or 'quarter'. ")
     }
   }
+  
+  if (per == "fortnight") {
+    if (is.null(.fortnights_per_yr)) {
+      .fortnights_per_yr <- 26
+      warning("`.fortnights_per_year` not set. Using 26 as default.")
+    }
+    if (.fortnights_per_yr == 26) {
```

Comment:
I think if `.fortnights_per_year` is numeric it need not be checked. If someone wants to use `.fortnights_per_yr = 25` (which it might be in some years) may we as well let him,